### ISSUE TITLE
volume control has no effect in calls

### ISSUE THREAD

**Issue by userj** [ubports/ubuntu-touch#1402]
### Illustrations

- Device: OnePlus3
- Channel: edge
- Build: 2020-03-22

### Steps to reproduce

* Get in a call
* Use the volume controls on the side of the phone

### Expected behavior

* Volume goes up and down

### Actual behavior

* Volume slider moves but there is no change to the volume

### Logfiles and additional information

* Related https://github.com/ubports/ubuntu-touch/issues/105#

**Comment by Flohack74** [comment]
@Vince1171 you might want to check if there are any *.acdb files in the original Android tree, in /system/etc. Note tht they also can be stored in subfolders. They might be essential for volume control and need to be symlinked to /etc

**Comment by ernesst** [comment]
Yes we have them : https://github.com/Halium/android_device_oneplus_oneplus3/commit/0d5c03850e5dcd2091d9a056cb21e6bce5246655

`root@ubuntu-phablet:~# ls -al /etc/ |grep acdb
lrwxrwxrwx    1 root root      20 May 22  1970 acdbdata -> /system/etc/acdbdata
root@ubuntu-phablet:~# ls -al /etc/acdbdata/MTP/
total 956
drwxr-xr-x. 2 root root   4096 Mar 23 23:56 .
drwxr-xr-x. 3 root root   4096 Mar 23 23:56 ..
-rw-r--r--. 1 root root  77149 Mar 23 23:56 MTP_Bluetooth_cal.acdb
-rw-r--r--. 1 root root  17435 Mar 23 23:56 MTP_General_cal.acdb
-rw-r--r--. 1 root root   9285 Mar 23 23:56 MTP_Global_cal.acdb
-rw-r--r--. 1 root root 328847 Mar 23 23:56 MTP_Handset_cal.acdb
-rw-r--r--. 1 root root   1503 Mar 23 23:56 MTP_Hdmi_cal.acdb
-rw-r--r--. 1 root root 167941 Mar 23 23:56 MTP_Headset_cal.acdb
-rw-r--r--. 1 root root 351721 Mar 23 23:56 MTP_Speaker_cal.acdb
root@ubuntu-phablet:~# 
`

**Comment by laborim** [comment]
I have the same problem

Device: OnePlus 3
Build: 2020-06-27/2

**Comment by userj** [comment]
Interesting note: in-call volume works on the GSI version.

**Comment by svenha** [comment]
Problem persists: OnePlus 3T, OTA-14 (stable)
No sound from the speaker (the small one for your ear, I don't know the right term, I fear) in calls.

For me, this is the most annoying bug for this device. Otherwise very good. Thank you, UBports!

**Comment by mattbel10** [comment]
I noticed this issue applies to my Pixel 3a as well when in a call and earphones are plugged in: volume modulation isn't working at all and the volume results set too high for my poor ears so I desperately need to lower the volume but I can't.
